src/coreclr/jit/lowerarmarch.cpp

```diff
@@ -1312,7 +1312,31 @@ GenTree* Lowering::LowerHWIntrinsic(GenTreeHWIntrinsic* node)
         case NI_AdvSimd_FusedMultiplyAddScalar:
             LowerHWIntrinsicFusedMultiplyAddScalar(node);
             break;
+        case NI_Sve_ConditionalSelect:
+        {
+            LIR::Use use;
+            bool     foundUse = BlockRange().TryGetUse(node, &use);
+
+            JITDUMP("lowering ConditionalSelect HWIntrinisic (before):\n");
+            DISPTREERANGE(BlockRange(), node);
+            JITDUMP("\n");
```

Comment:
This is going to no-op in most cases, right? Is it worth making the dump bigger for such no-ops?

Reply:
While we are improving `ConditionalSelect`, most likely lowering will change things and it really helped me in visualizing things while debugging. Will keep it for now and remove it once we stabilize.